### ISSUE TITLE
Unexpected error with `unit-no-unknown` on custom property variable with hex value

### ISSUE THREAD

**Issue by apascal** [stylelint/stylelint#1306]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug where an unexpected error is triggered when using custom variable with hexadecimal value starting with one or several leading numbers (eg. `#1a1a1a` or `#12b`).

Note: if the leading character is a letter, or the hex value is between quotes, or there are no letters at all, no error is triggered.

> Which rule, if any, is this issue related to?

`unit-no-unknown`

> What CSS is needed to reproduce this issue?

```css
:root {
  --bgColor: #1a1a1a;
}
```

> Which version of stylelint are you using?

`6.4.0` with stylelint-config-standard `7.0.0`
No error was triggered with stylelint `<6.4.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint *.css`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

It's related to CSS variables (https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables) (http://cssnext.io/features/#custom-properties-var)

> What did you expect to happen?

No error to be triggered.

> What actually happened (e.g. what warnings or errors you are getting)?

The following error was triggered:

```shell
test.css
2:14  Unexpected unknown unit "a1a1a"   unit-no-unknown
```

**Comment by evilebottnawi** [comment]
@apascal fixed in `6.4.1`, please update package.

**Comment by apascal** [comment]
Thx for the quick fix !
